notebooks/logging-examples/logging-basics.ipynb

```diff
@@ -6,234 +6,577 @@
+    "Each sample holds measurements of different qualities of the wines observed. We'll train on 75%\n",
+    "of the data and use the other 25% to make predictions using our model.\n",
     "\n",
-    "Let's shuffle the data and define our training and prediction subsets:"
+    "First we'll split the data to define our training and testing subsets."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 3,
    "metadata": {},
    "outputs": [],
    "source": [
-    "from datetime import datetime\n",
     "from sklearn.model_selection import train_test_split\n",
     "\n",
-    "random_state = int(datetime.utcnow().timestamp())\n",
-    "X_train, X_test, y_train, y_test = train_test_split(iris['data'], iris['target'], random_state=random_state)"
+    "\n",
+    "X_train, X_test, y_train, y_test = train_test_split(\n",
+    "    wine[\"data\"],\n",
+    "    wine[\"target\"],\n",
+    "    test_size=0.25,\n",
+    ")"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Let's define a method for fitting our model. We'll use sklearn's [Random Forest Classifier](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestClassifier.html)."
+    "Let's try to train a model using ``scikit-learn``'s\n",
+    "[random forest classifier](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestClassifier.html)!\n",
+    "\n",
+    "\n",
+    "**For this basic example, our goal will be to evaluate how the n_estimators parameter affects our results.**"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 4,
    "metadata": {},
    "outputs": [],
    "source": [
     "from sklearn.ensemble import RandomForestClassifier\n",
     "\n",
-    "def fit_classifier(n_estimators, n_features, random_state, X_train, y_train):\n",
-    "    rfc = RandomForestClassifier(n_estimators=n_estimators, random_state=random_state)\n",
+    "\n",
+    "def fit_and_score_classifier(X_train, X_test, y_train, y_test, n_estimators):\n",
+    "    rfc = RandomForestClassifier(n_estimators=n_estimators)\n",
     "    rfc.fit(X_train, y_train)\n",
     "    \n",
-    "    return rfc"
+    "    accuracy = rfc.score(X_test, y_test)\n",
+    "    \n",
+    "    return accuracy"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "RandomForestClassifier accepts `n_estimators` (the number of trees in the forest) as a **parameter**. What if we wanted to tune this **parameter** to see how it affected our output **metric**, accuracy?\n",
-    "\n",
-    "This is where `Rubicon`'s **experiment** logging comes in handy. Let's get `Rubicon` setup:"
+    "With ``rubicon``, we can create a **project** to store as many **experiments** as we'd\n",
+    "like to run. Each of these **experiments** can hold important metadata about your\n",
```

Comment:
Nit: it switches from `we` and `our` to `you` and `your` in this example, might want to switch all one way